_includes/sections/dns.html

```diff
@@ -285,6 +285,22 @@ <h4>Terms</h4>
     <li>DNSCrypt - An older yet robust method of encrypting DNS.</li>
   </ul>
 
+  <h4>How to Validate</h4>
+
+  <ul>
+    <li>DoH / DoT
+      <ul>
+        <li>Check <a href="https://www.dnsleaktest.com/">https://www.dnsleaktest.com/</a>.</li>
```

Comment:
I think this need additional information.

```suggestion
        <li>Check <a href="https://www.dnsleaktest.com/">https://www.dnsleaktest.com/</a> You should see something else than your ISP's DNS servers that is at least in the country of your chosen encrypted DNS server.</li>
```

I am not how good my suggestion is though. Also I am not sure how clear it is that Quad9 often appears as WoodyNet for people in the USA while in Finland they appear as TREX Exchange Services Oy where the node is hosted. My Keybase DNS leak test with my config file is possibly a good showcasing of this.

Reply:
Yeah, good point :thinking: the server location difference is due to Anycast, correct (which we call out in the table)? Maybe we should mention here in a warning or something about for Anycast providers, this may not be the most reliable check then...

Reply:
I'll push an update for this.